packages/sdk-codegen/src/python.gen.spec.ts

```diff
@@ -50,11 +50,17 @@ describe('python generator', () => {
     it('has a models prologue', () => {
       expect(gen.modelsPrologue('')).toEqual(`
 # NOTE: Do not edit this file generated by Looker SDK Codegen
-import attr
 import datetime
 import enum
 from typing import Any, MutableMapping, Optional, Sequence
 
+try:
+    from typing import ForwardRef  # type: ignore
+except ImportError:
+    from typing import _ForwardRef as ForwardRef  # type: ignore
+
+import attr
+
```

Comment:
need this in the prologue now that we use `ForwardRef` in some `__annotations__` in model declarations